styles/style.css

```
@@ -1,31 +1,601 @@
+@import url(https://fonts.bunny.net/css?family=merriweather:300,400,700,900|roboto:100,300,400,500);
+/* font-family: 'Merriweather', serif;*/
+/*font-family: 'Roboto', sans-serif;*/
 /*////////////////////////
     Table of Contents
     1. Global Styles
     2. Typography
-    3. Layout
-        3.1 Reusables
-        3.2 Other
-    4. Areas
-        4.1 Header
-        4.2 Introduction
-        etc.
-    5. Media Queries
-////////////////////////*/
+    3. Areas
+        3.1 Header
+        3.2 Introduction
+        3.3 Projects
+        3.4 Services
+        3.5 Contact Form
+        3.6 Footer
+ */
+    /*4. Media Queries
+       4.1 Header Mobile
+       4.2 Introduction Mobile
+       4.3 Projects Mobile
+       4.4 Services Mobile
+       4.5 Contact Form Mobile
+*/
+ /*////////////////////////*/
 
 /* ------------------------------- 1. Global Styles -------------------------------- */
+* {
+    Margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+:root {
+    --purple: #5722CD;
+    --blue: #4845E4;
+    --pink: #ED4385;
+    --black: #0C0C0C;
+    --light-purple: #F2F0FF;
+    --light-gray: #F3F6F9;
+    --gray: #7A7A7A;
+    --twidth: calc(100vw / 1 );
+    --theight: calc(var(--twidth) / 50);
+}
+
+.button {
+    background-color: var(--pink);
+    color: white;
+    border: transparent;
+    padding: 1.3rem 1.5rem;
+    min-width: 230px;
+    align-self: center;
+    text-transform: uppercase;
+    border-radius: 50px 50px;
```

Dit kun je ook korter schrijven
```suggestion
    border-radius: 50px;
```